### PR TITLE
setting-page

### PR DESCRIPTION
**Pull request by bhondgesanket768**

### CODE REVIEW

File: src/pages/configure/configure.jsx [hunk 1]
```diff
@@ -108,9 +112,10 @@ const Configure = () => {
 	return (
 		<div className="configure-page">
 			<Topbar showProjectSelector />
-			<div>
-				<Sidenav selectedItem="configure" />
-				<div className="page-content">
+			<Sidenav selectedItem="settings" />
+			<div className="page-content page-content--no-padding">
+				<SettingTabs activeKey='Environment Settings' projectID={projectID} />
+				<div style={{ paddingTop: 20 }}>
```
Comment (Saif-Ali-Kamal) [inline src/pages/configure/configure.jsx]: make some space in bottom after delete button

File: src/components/configure/whiteListedDomain.jsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react'
+import { Form, Select, Input, Button, Alert } from 'antd';
+
+
+const WhiteListedDomains = ({ form }) => {
+    const { getFieldDecorator } = form;
+    const { Option } = Select;
+    const children = [];
+    for (let i = 10; i < 36; i++) {
```
Comment (Jayesh333) [inline src/components/configure/whiteListedDomain.jsx]: Why are you doing this? In whitelisted domains there wont be any options. User will just type and enter and it should be added as a chip

File: src/pages/settings/general/General.jsx [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+
+    const handleModalCancel = () => {
+        //setRuleClicked("")
+        setModalVisibile(false)
+    }
+
+    const nameCopied = (e) => {
+        e.preventDefault();
+        setNameCopy("copied");
+        setTimeout(() => setNameCopy("copy"), 15000);
```
Comment (Jayesh333) [inline src/pages/settings/general/General.jsx]: 15 seconds is way long. Can we keep it 5 seconds?

File: src/pages/settings/general/General.jsx [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+                <Topbar showProjectSelector />
+                <div>
+                    <Sidenav selectedItem="settings" />
+                    <div className='page-content page-content--no-padding'>
+                        <SettingTabs activeKey='General' projectID={projectID} />
+                        <div className="db-tab-content">
+                            <h2>Credentials</h2>
+                            <Card style={{ display: "flex", justifyContent: "space-between" }}>
+                                <h3 style={{ wordSpacing: 6 }}><b>username </b>  noorainp <a onClick={nameCopied}>{nameCopy}</a></h3>
+                                <h3 style={{ wordSpacing: 6 }}><b>Access Key </b>  ************************* <a onClick={keyCopied}>{keyCopy}</a></h3>
```
Comment (Jayesh333) [inline src/pages/settings/general/General.jsx]: Nothing is actually getting copied here. Can we copy keep it to clipboard?

File: src/pages/settings/general/General.jsx [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+import client from "../../../client"
+import store from "../../../store";
+import history from "../../../history"
+
+function Settings() {
+    const { projectID, selectedDB } = useParams()
+
+    const dispatch = useDispatch()
+
+    const [ModalVisible, setModalVisibile] = useState(false)
```
Comment (Jayesh333) [inline src/pages/settings/general/General.jsx]: variable names in react are always camelCase

**Pull request merged**